src/bigquery.ts

```diff
@@ -328,10 +335,17 @@ export class BigQuery extends Service {
   }
 
   constructor(options: BigQueryOptions = {}) {
-    let apiEndpoint = 'https://bigquery.googleapis.com';
+    let universeDomain = 'googleapis.com';
+    const servicePath = 'bigquery';
+
+    if (options.universeDomain) {
+      universeDomain = BigQuery.sanitizeDomain(options.universeDomain);
+    }
 
     const EMULATOR_HOST = process.env.BIGQUERY_EMULATOR_HOST;
 
+    let apiEndpoint = `https://${servicePath}.${universeDomain}`;
```

Comment:
Should pass along the universeDomain param to the upper service class, same way as the apiEndpoint. This way the configurations is passed to the auth config.